app/views/directives/traffic-table.html

```diff
@@ -14,9 +14,9 @@
   <tbody ng-if="(portsByRoute | hashSize) == 0">
     <tr><td colspan="7"><em>{{emptyMessage || 'No routes or ports to show'}}</em></td></tr>
   </tbody>
-  <tbody ng-if="(portsByRoute | hashSize) > 0">
-    <tr ng-repeat-start="(routeName,ports) in portsByRoute" style="display:none;"></tr>
-    <tr ng-repeat="port in ports" ng-if="routeName !== ''">
+  <tbody ng-if="(portsByRoute | hashSize) > 0" ng-repeat-start="(routeName,ports) in portsByRoute" style="display:none;"></tbody>
+  <tbody ng-repeat="port in ports" ng-if="routeName !== ''">
```

Comment:
@spadgett The reason why the double border occurs at the top of the table, is because I am "nesting" two `ng-repeat`s using sibling `tbody` elements. Although the first one is hidden with `display: none`, it is causing the second one to have a `2px` border when it otherwise would not due to it normally being the [first `tbody` element](https://github.com/openshift/origin-web-console/blob/master/app/styles/_tables.less#L123).